tensorflow_addons/custom_ops/image/python/dense_image_warp_test.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for dense_image_warp."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import math
+
+import numpy as np
+import tensorflow as tf
+
+from tensorflow.python.framework import test_util as tf_test_util
+from tensorflow_addons.custom_ops.image.python import dense_image_warp as dense_image_warp_ops
+
+
+class DenseImageWarpTest(tf.test.TestCase):
+    def setUp(self):
+        np.random.seed(0)
+
+    @tf_test_util.run_all_in_graph_and_eager_modes
```

Comment:
Oh, it seems that we make a mistake. We should use `run_in_graph_and_eager_modes` for method